CHANGELOG.adoc

```diff
@@ -1,11 +1,22 @@
-== iZettle API Changelog
+== Zettle API Changelog
+.**April 30, 2021** `Pusher API`
+[%collapsible]
+====
+**Pusher API documentation Improvement**
+
+The Pusher API documentation is improved as follows:
+
+- Improved the list of supported events.
+- Improved API reference.
+- Added user guides and troubleshooting for using the Pusher API.
+====
```

Comment:
Lets rephrase this a little bit:
Improved API reference document.
Added the list of only supported events in the API reference document.
Added the user guides and troubleshooting documents.